Sources/Misc/Obsoletions.swift

```diff
@@ -719,3 +719,16 @@ public extension Purchases {
     enum PeriodType {}
 
 }
+
+extension EntitlementInfo {
+
+    @available(iOS, obsoleted: 1, message: "Use `CustomerInfo.rawData`")
+    @available(tvOS, obsoleted: 1, message: "Use `CustomerInfo.rawData`")
+    @available(watchOS, obsoleted: 1, message: "Use `CustomerInfo.rawData`")
+    @available(macOS, obsoleted: 1, message: "Use `CustomerInfo.rawData`")
+    @available(macCatalyst, obsoleted: 1, message: "Use `CustomerInfo.rawData`")
+    public var rawData: [String: Any] {
+        fatalError()
```

Comment:
this honestly doesn't feel great - we _just_ added the feature and we're already obsoleting it

Reply:
also, this is an obsoletion with a fatal error, this would mean a major version

Reply:
naive question: why can't this be loaded into the EntitlementInfo as `rawData["subscriber"]["entitlements"]`?

Reply:
Yeah looking back this definitely doesn't feel like a good tradeoff, hopefully we can find an easy way to do this.
I think I just thought of a way, standby! I'm holding on merging #1496 because this is kind of a blocker.

Reply:
Okay no more deprecation :) And I added some extra data to the test fixture, and verified that it gets decoded as part of the raw data :)